src/components/UI/DrawerItems/index.tsx

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
-import AppBar from '@mui/material/AppBar';
+import * as React from 'react'; // Import React Components, hooks
```

Comment:
Its best practice to avoid importing * anything since it takes up too much memory for things your arnt even using so a better way would be to only import the exact things your are using in this case just React, and useState.